### DISCUSSION TITLE
ackTimeoutRedeliveryBackoff not support "SubscriptionType.Exclusive"?

### DISCUSSION THREAD

**Discussion by startjava** [apache/pulsar#19600] [Q&A] [answered]
### Search before asking

- [X] I searched in the [issues](https://github.com/apache/pulsar/issues) and found nothing similar.

### Version

@Bean
public Consumer createPulsarConsumer_myTopic2() throws PulsarClientException {
    Consumer consumer = pulsarClient.newConsumer().topic(myTopic2).subscriptionType(SubscriptionType.Exclusive).subscriptionName(myTopic2_subscriptionName).messageListener(myMessageListener2).ackTimeout(10, TimeUnit.SECONDS)
            .ackTimeoutRedeliveryBackoff(MultiplierRedeliveryBackoff.builder()
                    .minDelayMs(1000)
                    .maxDelayMs(60 * 1000)
                    .multiplier(2)
                    .build()).subscribe();
    return consumer;
}

RedeliveryBackoff time alway 12s，But use Shared is OK !
it maybe bug ?

### Minimal reproduce step

?

### What did you expect to see?

?

### What did you see instead?

?

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

**Comment by startjava** [comment]
@codelipenghui

**Comment by tisonkun** [comment, marked as answer]
From [the document](https://pulsar.apache.org/docs/2.11.x/concepts-messaging/#delayed-message-delivery), it writes:

> Delayed message delivery only works in the Shared subscription type. In the Exclusive and Failover subscription types, the delayed message is dispatched immediately.
